R/adorn_totals.R

```diff
@@ -133,9 +144,28 @@ adorn_totals <- function(dat, where = "row", fill = "-", na.rm = TRUE, name = "T
         dplyr::select_if(is.numeric) %>%
         dplyr::transmute(Total = rowSums(., na.rm = na.rm))
       
-      dat[[name]] <- row_totals$Total
+      dat[[name[2]]] <- row_totals$Total
     }
     
+    # pure dplyr alternative:
+    # if ("col" %in% where) {
+    #   # Add totals col
+    #   dat <- dat %>%
+    #     dplyr::select(cols_to_total) %>%
+    #     dplyr::select_if(is.numeric) %>%
+    #     dplyr::transmute(name[2] = rowSums(., na.rm = na.rm)) %>% 
+    #     dplyr::bind_cols(dat, .)
+    # }
+    
+    # dplyr >= 1.0 version:
+    # if ("col" %in% where) {
+    #   # Add totals col
+    #   dat <- dat %>%
+    #     dplyr::rowwise() %>% 
+    #     dplyr::mutate(name[2] = sum(c_across(all_of(cols_to_total) & where(is.numeric)), na.rm = na.rm)) %>% 
+    #     dplyr::ungroup()
```

Comment:
Thanks for these!  I tried the dplyr >= 1.0 one for a while but had failing tests so just for the sake of time I kept what I had.  I got as far as changing your line 165 to `mutate("{name[2]}" := sum(c_across` - the `mutate(name[2]` wasn't working for me.

Reply:
No need to work on it, I think status quo is okay - just sharing my experience in case we come back to this someday.